### PR TITLE
NHWC  optimizations (MKL)

### PR DESCRIPTION
**Pull request by agramesh1**
Update MKL link to point to latest version (mklml_lnx_2018.0.20170720) . Also undoes workarounds previously added for NHWC conversions.

### CODE REVIEW

**Comment by rmlarsen** [comment]
@tensorflow-jenkins test this please

**Comment by rmlarsen** [comment]
It looks like the build failed to download the MKL package. Is there perhaps a typo in the URL?

**Comment by rmlarsen** [comment]
@tensorflow-jenkins test this please

**Comment by agramesh1** [comment]
@rmlarsen It looks the latest version of MKL  - github.com/01org/mkl-dnn/releases/download/v0.9/mklml_lnx_2018.0.20170720.tgz  is not mirrored on mirror.bazel.build. How can I get it mirrored?

**Comment by rmlarsen** [comment]
@andydavis1 @martinwicke what it the process for mirroring the Intel packages?

**Comment by martinwicke** [comment]
@jart is master of the mirror dimension.

**Comment by rmlarsen** [comment]
Hi @jart would you please mirror the new Intel MKL package, then assign back to me? Thanks!

**Comment by agramesh1** [comment]
@jart any updates?

File: tensorflow/workspace.bzl [hunk 1]
```diff
@@ -153,11 +153,11 @@ def tf_workspace(path_prefix="", tf_repo_name=""):
   mkl_repository(
       name = "mkl",
       urls = [
-          "http://mirror.bazel.build/github.com/01org/mkl-dnn/releases/download/v0.7/mklml_lnx_2018.0.20170425.tgz",
-          "https://github.com/01org/mkl-dnn/releases/download/v0.7/mklml_lnx_2018.0.20170425.tgz",
+          "http://mirror.bazel.build/github.com/01org/mkl-dnn/releases/download/v0.9/mklml_lnx_2018.0.20170720.tgz",
+          "https://github.com/01org/mkl-dnn/releases/download/v0.9/mklml_lnx_2018.0.20170720.tgz",
       ],
       sha256 = "3cc2501fb209e1fd0960a5f61c919438f9619c68a644dcebf0fdf69b07460c57",
```
Comment (jart) [inline tensorflow/workspace.bzl]: It appears this should be: 57ba56c4c243f403ff78f417ff854ef50b9eddf4a610a917b7c95e7fa8553a4b

File: tensorflow/workspace.bzl [hunk 2]
```diff
@@ -153,11 +153,11 @@ def tf_workspace(path_prefix="", tf_repo_name=""):
   mkl_repository(
       name = "mkl",
       urls = [
-          "http://mirror.bazel.build/github.com/01org/mkl-dnn/releases/download/v0.7/mklml_lnx_2018.0.20170425.tgz",
-          "https://github.com/01org/mkl-dnn/releases/download/v0.7/mklml_lnx_2018.0.20170425.tgz",
+          "http://mirror.bazel.build/github.com/01org/mkl-dnn/releases/download/v0.9/mklml_lnx_2018.0.20170720.tgz",
+          "https://github.com/01org/mkl-dnn/releases/download/v0.9/mklml_lnx_2018.0.20170720.tgz",
```
Comment (jart) [inline tensorflow/workspace.bzl]: This has now been mirrored.

**Review by jart — DISMISSED** [review]

**Comment by rmlarsen** [comment]
@jart Thanks!

**Comment by rmlarsen** [comment]
@agramesh1 Could you please check the hash as @jart suggests?
@tensorflow-jenkins test this please

**Comment by agramesh1** [comment]
@rmlarsen updated files as requested by @jart , thanks.

**Comment by rmlarsen** [comment]
@tensorflow-jenkins test this please

**Review by rmlarsen — APPROVED** [review]

**Comment by rmlarsen** [comment]
Thanks for the update!

**Pull request merged**